anitya/templates/project_new.html

```diff
@@ -119,21 +134,31 @@
     if (str == 'Sourceforge'){
         $("label[for='version_url']").text('Sourceforge name');
         $('#version_url').after(
-        "<p>Use this field if the name used on the rss page of the project "
+        "<p id='sourceforge_version_url_instr'>Use this field if the name used on the rss page of the project "
         + "differs from the name of the project. </p>" );
         $('#version_url_row').show();
     };
     if (str == 'custom'){
-      $('#regex_row').show();
       $('#version_url_row').show();
       $('#insecure_row').show();
       $('#regex').attr('placeholder', 'Leave empty for default regex');
+
+      if ($('#name').val()){
+        $('#regex_row').show();
+        $('#default_regex_row').show();
+      }
```

Comment:
We could show the default regex all the time and adjust it upon change in the ``#name`` field :)